src/main/java/com/team1816/lib/subsystems/Infrastructure.java

```diff
@@ -23,7 +24,7 @@
     private Infrastructure() {
         super("Infrastructure");
         if (Constants.kPCMId >= 0) {
```

Comment:
You need to remove this if check for the constants it is bypassing the factory.  I would kust remove the entire constant at this point.